packages/app/src/cli/models/extensions/specifications/transform/app_config_webhook.ts

```diff
@@ -1,17 +1,22 @@
+import {AppConfigurationWithoutPath, CurrentAppConfiguration} from '../../../app/app.js'
 import {WebhooksConfig, NormalizedWebhookSubscription, WebhookSubscription} from '../types/app_config_webhook.js'
 import {deepCompare, deepMergeObjects, getPathValue} from '@shopify/cli-kit/common/object'
 
-export function transformFromWebhookConfig(content: object) {
+export function transformFromWebhookConfig(content: object, appConfiguration: AppConfigurationWithoutPath) {
   const webhooks = getPathValue(content, 'webhooks') as WebhooksConfig
   if (!webhooks) return content
 
   const webhookSubscriptions = []
   // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
I've added to https://github.com/Shopify/develop-app-management/issues/1797

Maybe we could change https://typescript-eslint.io/rules/naming-convention/#ignore-destructured-names -- if you're destructuring its not really your fault that the names are "wrong".